sdk/cosmosdb/cosmos/package.json

```diff
@@ -94,6 +94,7 @@
     "@azure/core-auth": "^1.3.0",
     "@azure/core-rest-pipeline": "^1.2.0",
     "@azure/core-tracing": "^1.0.0",
+    "@azure/identity": "^2.0.1",
```

Comment:
identity should never be a direct runtime dependency for any SDK. It should be brought in by the consumer's package.json

Reply:
reverted.